pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -251,6 +251,9 @@ func changefeedPlanHook(
 				if err := changefeedbase.ValidateTable(targets, table); err != nil {
 					return err
 				}
+				for _, warning := range changefeedbase.WarningsForTable(targets, table) {
+					p.BufferClientNotice(ctx, pgnotice.Newf(warning.Error()))
+				}
```

Comment:
This is probably okay, but I just want to point out that this warning will not be visible to core change feed until change feed terminates